### PR TITLE
Accessibility for cardslist

### LINKED ISSUE #940: Accessibility: Support accessibility for List of Cards screen
Make sure that accessibility is supported for:
`delern/flutter/lib/views/cards_list/cards_list.dart`
             

**Doc**: https://flutter.dev/docs/development/accessibility-and-localization/accessibility
**Tooltip - Widget of the week**: https://www.youtube.com/watch?v=EeEfD5fI-5Q

### PR DESCRIPTION
**Pull request by jddeep**
Accessibility for cardslist.

Fixes #940

### CODE REVIEW

**Comment by jddeep** [comment]
@ksheremet Please review and merge.

**Comment by jddeep** [comment]
@ksheremet Please review and merge.

File: flutter/lib/views/cards_list/cards_list.dart [hunk 1]
```diff
@@ -45,8 +46,24 @@ class _CardsListState extends State<CardsList> {
...
   @override
   Widget build(BuildContext context) => Scaffold(
-        appBar: SearchBarWidget(
-            title: widget.deck.name, search: _searchTextChanged),
+        appBar: searchClicked
+            ? SearchBarWidget(
+                title: widget.deck.name,
+                search: _searchTextChanged,
+              )
+            : AppBar(
```
Comment (ksheremet) [inline flutter/lib/views/cards_list/cards_list.dart]: @jddeep 
Please do not duplicate code. Could you please fix styles in SearchBarWidget?

Comment (jddeep) [reply flutter/lib/views/cards_list/cards_list.dart]: @ksheremet What's wrong? I made changes and flutter formatted after that.

Comment (jddeep) [reply flutter/lib/views/cards_list/cards_list.dart]: @ksheremet Can you be specific like what you want here?

Comment (ksheremet) [reply flutter/lib/views/cards_list/cards_list.dart]: @jddeep 
No need in this code
```dart
AppBar(
      backgroundColor: Colors.green[800],
             actions: <Widget>[
                  IconButton(
                      icon: const Icon(Icons.search),
                      onPressed: () {
                        setState(() {
                          searchClicked = true;
                        });
                      },
                      tooltip: 'Search')
                ],
              )
```

You can go to ```flutter/lib/views/helpers/search_bar_widget.dart``` and setup  ```backgroundColor: Colors.green[800]``` there.

Comment (jddeep) [reply flutter/lib/views/cards_list/cards_list.dart]: Alright @ksheremet Done the changes. Please review.

**Comment by ksheremet** [comment]
@jddeep 
error • The argument for the named parameter 'tooltip' was already specified • lib/views/helpers/search_bar_widget.dart:76:11 • duplicate_named_argument

**Comment by jddeep** [comment]
@ksheremet removed. Please review.

**Pull request merged**